src/main/java/com/code_intelligence/jazzer/driver/OfflineInstrumentor.java

```diff
@@ -55,11 +55,14 @@ public static boolean instrumentJars(List<String> jarLists) {
 
     List<String> errorMessages = new ArrayList<>();
     for (String jarPath : jarLists) {
+      String jarBaseName =
+          jarPath.substring(jarPath.lastIndexOf('/') + 1, jarPath.lastIndexOf('.'));
+
       try {
         Log.info("Instrumenting jar file: " + jarPath);
         errorMessages = createInstrumentedClasses(jarPath);
         createInstrumentedJar(
-            jarPath, Opt.dumpClassesDir, Opt.dumpClassesDir + ".instrumented.jar");
+            jarPath, Opt.dumpClassesDir + "/" + jarBaseName, jarBaseName + ".instrumented.jar");
```

Comment:
Use `File.separator` for system independent separator character, or build up the path as `File` or `Path`.